source/Weapon.cpp

```diff
@@ -79,6 +79,7 @@ void Weapon::LoadWeapon(const DataNode &node)
 		}
 		else
 		{
+<<<<<<< HEAD
```

Comment:
I don't think this is supposed to be here :)

Reply:
Also, I think there are now other uses of `outfit->Velocity()` for aiming and such that should be switched to use `WeightedVelocity()`